addons/l10n_in_website_sale/__manifest__.py

```diff
@@ -0,0 +1,9 @@
+{
+    'name': "Indian eCommerce",
+    'description': "Indian Localization E-Commerce Module",
+    'version': '1.0',
+    'countries': ['in'],
+    'depends': ['l10n_in_gstin_status', 'website_sale'],
+    'installable': True,
+    'license': 'LGPL-3',
```

Comment:
```suggestion
    'installable': True,
    'author': 'Odoo S.A.',
    'license': 'LGPL-3',
```
Don't forget to add author, in master it's a mandatory to define `author` or the runbot will fail